stubs/algopy-stubs/_state.pyi

```diff
@@ -2,7 +2,7 @@ import typing
 
 from algopy import Account, Bytes, BytesBacked, UInt64
 
-_TState = typing.TypeVar("_TState", bound=UInt64 | Bytes | BytesBacked)
+_TState = typing.TypeVar("_TState", bound=UInt64 | Bytes | BytesBacked | bool)
```

Comment:
This makes me release we might need a `UInt64Backed` for things like Asset and Application, or at least include them in this TypeVar

As currently this works

`self.some_asset = Asset(....)`

But this won't
```
self.some_asset = GlobalState(Asset)
self.some_asset.value = Asset(...)
```

Reply:
I think we either expand the type var bounds or just drop the bounds.

Not sure a `UInt64Backed` makes sense in the same way that `BytesBacked` does.